shared/js/background/dnr-click-to-load.js

```diff
@@ -69,6 +69,51 @@ async function generateDnrAllowingRules (tab, ruleAction) {
     return allowingRules
 }
 
+/**
+ * Find the enabled Click to Load rule actions for the given tab.
+ * Note: Take care to ensure wait for the extension configuration to be ready
+ *       first.
+ * @param {import('./classes/tab.es6')} tab
+ * @return {string[]}
+ */
+export function getDefaultEnabledClickToLoadRuleActionsForTab (tab) {
+    // Click to Load feature isn't supported or is disabled for the tab.
+    if (!utils.getClickToPlaySupport(tab)) {
+        return []
+    }
+
+    const clickToLoadSettings =
+        tdsStorage?.config?.features?.clickToPlay?.settings
+
+    // Click to Load configuration isn't ready yet.
+    if (!clickToLoadSettings) {
+        console.warn('Click to Load configuration not ready yet, skipped.')
+        return []
+    }
+
+    const enabledRuleActions = []
+    const { parentEntity } = tab.site
+
+    for (let [entity, { ruleActions, state }] of Object.entries(clickToLoadSettings)) {
+        // No rule actions, or entity is disabled.
+        if (!ruleActions || ruleActions.length === 0 || state !== 'enabled') {
+            continue
+        }
+
+        // TODO: Remove this workaround once the content-scope-scripts and
+        //       privacy-configuration repositories have been updated.
+        if (entity === 'Facebook') entity = 'Facebook, Inc.'
```

Comment:
I changed this deliberately so that when update "Facebook" => "Facebook, Inc." in the config + content-scope-scripts the logic won't break. It's annoying doing the `=== 'Facebook'` comparison in the loop, but it seemed worth it.